src/block.js

```diff
@@ -24,9 +51,22 @@ export default function Block( { currentTotal, freeShippingFrom } ) {
 
 	return (
 		<div className="wc-free-shipping-progress-bar">
-			<div id="message">{ message }</div>
-			<div id="progressBar">
-				<div id="progress" style={ divStyle }></div>
+			<div className={ classnames( 'message', messageClasses ) }>
+				{ message }
+			</div>
+			<div
+				className={ classnames(
+					'progress-container',
+					progressBorderClasses
+				) }
+			>
+				<div
+					className={ classnames(
+						'progress-bar',
+						progressBackgroundClasses
+					) }
+					style={ progressBarStyle }
+				></div>
```

Comment:
Your changes should be immediately reflected on the editor, they're not.
![image](https://user-images.githubusercontent.com/6165348/135482353-7e70490f-5f76-4199-9d00-6b0c5295741c.png)

Reply:
On my end, the changes take place immediately:

![Screen Capture on 2021-09-30 at 22-39-10](https://user-images.githubusercontent.com/3323310/135487294-6f187287-9c36-4c19-b238-c61a157528d0.gif)